README.md

```diff
@@ -1,78 +1,50 @@
-- Encapsulated in one object
-- No browser sniffing for critical functionality
-- Runs under ES5 strict mode
-- Runs on a server under [Node.js](http://nodejs.org/) (active stable releases and latest of current) (see [Node.js limitations](https://github.com/kangax/fabric.js/wiki/Fabric-limitations-in-node.js))
-- Follows [Semantic Versioning](http://semver.org/)
-
-### Supported browsers
-
-- Firefox 4+
-- Safari 5+
-- Opera 9.64+
-- Chrome (all versions)
-- Edge (chromium based, all versions)
-- IE11 and Edge legacy, not supported. Fabric up to 5.0 is written with ES5 in mind, but no specific tests are run for those browsers.
-
-You can [run automated unit tests](http://fabricjs.com/test/unit/) right in the browser.
-
-### History
-
-Fabric.js started as a foundation for design editor on [printio.ru](http://printio.ru) — interactive online store with ability to create your own designs. The idea was to create [Javascript-based editor](http://printio.ru/ringer_man_tees/new), which would make it easy to manipulate vector shapes and images on T-Shirts. Since performance was one of the most critical requirements, we chose canvas over SVG. While SVG is excellent with static shapes, it's not as performant as canvas when it comes to dynamic manipulation of objects (movement, scaling, rotation, etc.). Fabric.js was heavily inspired by [Ernest Delgado's canvas experiment](http://www.ernestdelgado.com/public-tests/canvasphoto/demo/canvas.html). In fact, code from Ernest's experiment was the foundation of an entire framework. Later, Fabric.js grew into a collection of distinct object types and got an SVG-to-canvas parser.
+[![Bower version](https://badge.fury.io/bo/fabric.svg)](http://badge.fury.io/bo/fabric)
 
-### Installation Instructions
+<br/>
 
-<h3 id="bower-install">Install with bower</h3>
 
-    $ bower install fabric
+Using Fabric.js, you can create and populate objects on canvas; objects like simple geometrical shapes — rectangles, circles, ellipses, polygons, or more complex shapes consisting of hundreds or thousands of simple paths. You can then scale, move, and rotate these objects with the mouse; modify their properties — color, transparency, z-index, etc. You can also manipulate these objects altogether — grouping them with a simple mouse selection. Fabric.js will also allow you to convert an SVG image into JavaScript data that can be used for putting it onto the `<canvas>` element.
 
-<h3 id="npm-install">Install with npm</h3>
+<br />
 
-Note: If you are using Fabric.js in a Node.js script, you will depend from [node-canvas](https://github.com/Automattic/node-canvas).`node-canvas` is an html canvas replacement that works on top of native libraries.
-Please follow the instructions located [here](https://github.com/Automattic/node-canvas#compiling) in order to get it up and running.
+<a href="http://fabricjs.com/fabric-intro-part-1">Introduction</a>
+<span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+<a href="http://fabricjs.com/docs/">Docs</a>
```

Comment:
I don't think this part is needed, github parses the page and creates a navbar

Reply:
[https://github.com/reactchartjs/react-chartjs-2](https://github.com/reactchartjs/react-chartjs-2) was my "inspiration" here. 😄 I didn't know that Github creates navbar somewhere. Where I can find it?

Reply:
In the README page.
![צילום מסך 2022-05-17 233113](https://user-images.githubusercontent.com/34343793/168904786-a4a69179-d0c3-4ce3-8b88-9009e625032e.png)